src/CurrentFileVersionCompatibilityTests/packages.config

```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Axe.Windows" version="0.3.1-prerelease" targetFramework="net472" />
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net472" />
   <package id="System.Buffers" version="4.5.0" targetFramework="net471" />
   <package id="System.Collections.Immutable" version="1.6.0" targetFramework="net471" />
   <package id="System.Memory" version="4.5.3" targetFramework="net471" />
   <package id="System.Numerics.Vectors" version="4.5.0" targetFramework="net471" />
   <package id="System.Runtime.CompilerServices.Unsafe" version="4.6.0" targetFramework="net471" />
+  <package id="Newtonsoft.Json" version="12.0.2" targetFramework="net471" />
```

Comment:
CurrentFileVersionCompatibilityTests needs to remain with 9.0.1, since that's what is needed for the version of Axe.Windows that we're using here